jmxterm/Dockerfile.deb9

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright 2017 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG DOCKER_UPSTREAM_REGISTRY
+ARG DOCKER_TAG
+
+FROM ${DOCKER_UPSTREAM_REGISTRY}confluentinc/cp-base-new:${DOCKER_TAG}
+
+MAINTAINER partner-support@confluent.io
+LABEL io.confluent.docker=true
+ARG COMMIT_ID=unknown
+LABEL io.confluent.docker.git.id=$COMMIT_ID
+ARG BUILD_NUMBER=-1
+LABEL io.confluent.docker.build.number=$BUILD_NUMBER
+
+WORKDIR /opt
+
+RUN wget http://downloads.sourceforge.net/project/cyclops-group/jmxterm/1.0-alpha-4/jmxterm-1.0-alpha-4-uber.jar -O /opt/jmxterm-1.0-alpha-4-uber.jar
```

Comment:
Please store this in the `confluent-packaging-tools` S3 Bucket - Add some sha256 verification as well please.

Reply:
@andrewegel Why do we need to do sha256 verification? We don't do that for any of the other resources that we install from that S3 bucket? Thanks for catching this though, I am moving that to an S3 bucket.

Reply:
> Why do we need to do sha256 verification? 

I'll change your question to *why should* - the answer is to prevent an insider from installing a back door - At the moment, anyone at confluent can just upload to this bucket, and override a key with say their backdoor jar, and no one would be the wiser unless someone looked through AWS S3 access logs. 

Adding SHA256 verification inside code here adds a layer of protection, as now the SHA256 of the artifact is in code, and we'd seen an insider changing that value in git.

> We don't do that for any of the other resources that we install from that S3 bucket?

I won't block shipping your change on implementing SHA256 validation (just host it in our S3 bucket) - But I do think its a worthy task to go back and fix. Or maybe we add 2FA S3 Bucket Deletes.

Reply:
@andrewegel Sounds like something good we can start doing. Since we do this type of thing in a lot of our images it seems like we should put together some best practice around this and a project to refactor our existing builds to start doing this. I will create a Jira ticket and cc you on it so you can add your thoughts.

Reply:
https://confluentinc.atlassian.net/browse/ST-3786